mw_organisation_activity.go

```diff
@@ -134,7 +126,7 @@ func (k *OrganizationMonitor) ProcessRequestOffThread(r *http.Request) (error, i
 	return nil, 200
 }
 
-func (k *OrganizationMonitor) AllowAccessNext(orgChan chan bool, r *http.Request) {
+func (k *OrganizationMonitor) AllowAccessNext(orgChan chan bool, path string, IP string, r *http.Request) {
 
```

Comment:
If IP and path are independent of the request then it makes sense to add them the function signature but if they are always part of it then why not then why did you add them (they can be set locally in the function, unless I misunderstood the purpose)

Reply:
Because the request object was causing a read panic later in the stack - see the convo in slack